### PR TITLE
Add some tests

### LINKED ISSUE #200: Add happy path tests for gcputils
### TL;DR

We are missing some test coverage. Add tests for PROJECT_ID, GOOGLE_PROJECT, and GOOGLE_CLOUD_PROJECT environment variables.

### Detailed design

_No response_

### Alternatives considered

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by shan4993**
resolves #200

### CODE REVIEW

File: gcputil/gcputil_test.go
Comment (yolocs) [inline gcputil/gcputil_test.go:1]: We should add sad-path tests for this package

File: gcputil/gcputil_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+package gcputil
+
+import (
+	"context"
+	"os"
+	"testing"
+)
+
+func TestProjectID(t *testing.T) {
+				t.Parallel()
```
Comment (yolocs) [inline gcputil/gcputil_test.go:23]: Fix the lint issue

File: gcputil/gcputil_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		{
+			name: "success_google_project",
+			env: map[string]string{
+				"GOOGLE_PROJECT":       "googleProject",
+				"GOOGLE_CLOUD_PROJECT": "googleCloudProject",
+			},
+			want: "googleProject",
+		},
+		{
+			name: "success_google_cloud_projecttt", // name typo, suggest fix in UI
```
Comment (yolocs) [inline gcputil/gcputil_test.go:48]: This is a typo
```suggestion
			name: "success_google_cloud_project", // name typo, suggest fix in UI
```

**Review by yolocs — CHANGES_REQUESTED** [review]
A few minor issues. @sethvargo

File: gcputil/gcputil_test.go
Comment (shan4993) [reply gcputil/gcputil_test.go:1]: lets do this as a follow up

Comment (shan4993) [reply gcputil/gcputil_test.go:1]: created #202

**Review by yolocs — APPROVED** [review]

**Pull request merged**